### ISSUE TITLE
Marker function usage used to be simpler :(

### ISSUE THREAD

**Issue by sinedied** [biesbjerg/ngx-translate-extract#189]
First I would like to thank you for maintaining this package!

I just discovered thanks to a bug reported on my downstream project that you removed the `--marker` flag, which IMO was way more simpler and straighforward to use than your new solution.

Having to install an additional package (our node_modules are already bloated as they are...) for a noop function tickles me a bit, and also force you to use it and alias the import name *everytime* if we want to change it. With the previous solution we could just define our own noop marker function (and even make it global) and that was it.

Just for the sake or removing one flag from the cli, it doesn't seem worth of the additional bloat in code and usage, and seeing some of the related issues it also does not seem as straighforward as it was.

Was there any blocking issue with the previous solution that made you go towards this change?

**Comment by sinedied** [comment]
I tried upgrading my codebase, the new mandatory package for JS strings extraction don't make things simple.

Trying to re-export the aliasing to avoid dodgy search & replace in codebase with:
```ts
export { marker as extract } from '@biesbjerg/ngx-translate-extract-marker';
```
Doesn't work.

Not really nice, especially as migrating to the new import means that I have to manually update all the imports now, hoping that it won't change again.

I get that maintaining an OSS project is not easy (I have a few on my side) especially with all that kind of complaints, but it would feel nice at need to have a bit of explanation about why you made this change and what benefits it brings. Spending hours to fix a codebase that used to be fine is not fun 😞

**Comment by votdev** [comment]
I have to agree to @sinedied, the latest approach is not optimal, it forces me to add `import { marker } from '@biesbjerg/ngx-translate-extract-marker';` in every file i want to use this helper function. Using my own function like it was possible in the previous version looks better to me. Please consider to re-enable the feature to define my own marker via `--marker` command line argument.
